puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/pipeline/foreman-release.yaml

```diff
@@ -60,10 +60,7 @@
       - 'foreman-plugins-{version}-rpm-pipeline'
     empty: ''
     version:
-      - '1.21'
-      - '1.22'
-      - '1.23'
-      - '1.24'
+      !include: ../../includes/foreman_versions.yaml.inc
```

Comment:
Thoughts on this approach to keep the jobs split out but achieve the single version location? See https://github.com/theforeman/foreman-infra/pull/1213/files#diff-ac0a597bb541cb1ebc55f7e2c6ab538dR27 as well

Reply:
I like. I'd like it even more if we could get rid of the odd nighly jobs, but that's not possible at the moment as far as I see it.

Reply:
I dislike it. There's already a lot of files and I liked to have a single file that defines all the release jobs.

Reply:
but the jobs are still in foreman-release.yml, just the versions are external so that they can be reused?

Reply:
Right now there's 2 places that use it, but I'd argue that everything in `foreman-packaging-rpm-release.yaml` should also be in `foreman-release.yaml`. Then there's just one place and you can clearly see what it impacts.

Reply:
Right, so I somehow thought this would move the jobs to one file and the version definition to another file. I would *prefer* the jobs in one file too, but as strongly as Ewoud ;)